script.js

```diff
@@ -0,0 +1,133 @@
+
+//1.Write a console.log statement saying "Hello World!" for each language that you know.
+console.log('Hello World!'); // English
+console.log('Merhaba Dunya!'); //Turkish
+console.log('Marhaba Dunia!') //Zazaki
+
+
+//2.Consider the following code:
+console.log('I\'m awesome');
+
+//3.Declare a variable age and initialize it with an integer, using these exact steps:
+let x;
+console.log('the value of age will be: 30');
+console.log(x);
+x=30;
+console.log('the value of age will be: 30');
+console.log(x);
+
+//4.Declare a variable name and assign a string to it.
+
+let y ='butterfly';
+console.log('the value of my string will be:bird');
+console.log(y);
+y='bird';
+console.log('the value of my string will be:bird');
+console.log(y);
+
+
+//5.How do you round the number 7.25, to the nearest integer (i.e., whole number)?
+
+let number=7.25;
+console.log(number);// prints 7.25
+
+let roundedNumber=Math.round(number);
+console.log(number); // prints 7
+
+if(roundedNumber < number){
```

Comment:
Good work here, but make sure you declare your variable `result` with the `let` keyword, i.e. `let result;`. Otherwise, it'll automatically become a global variable (https://www.w3schools.com/js/js_scope.asp)